Makefile

```diff
@@ -74,7 +74,7 @@ cmd: cmd-gen
 
 .PHONY: cmdx
 CMDX_PLATFORMS = "windows/amd64" "darwin/amd64" "linux/amd64"
-CMDX_CMDS = "passport-scanner"
+CMDX_CMDS = "passport-scanner" "passport" "privatedata-exchange"
```

Comment:
We will not be able to build these tools ("passport" "privatedata-exchange") for all platforms, because they use CGO.